api/src/main/java/io/opencensus/trace/CurrentSpanUtils.java

```diff
@@ -74,4 +100,73 @@ public void close() {
       }
     }
   }
+
+  private static final class RunnableInSpan implements Runnable {
+    private final Span span;
```

Comment:
nit since the type is private, everything else isn't accessible. Removing redundant visibility modifiers results in smaller bytecode iirc